src/js/components/Form/stories/UnsolicitedFeedback.js

```diff
@@ -0,0 +1,137 @@
+        <Button
+          onClick={onOpen}
+          margin={{ vertical: 'medium', horizontal: 'medium' }}
+          elevation="large"
+          color="purple!"
+          label="Submit Feedback"
+          a11yTitle="This button launches a modal to give feedback."
+          primary
+          alignSelf="start"
+        />
+      </PositionedFeedbackBox>
+      {open && (
+        <Layer modal={false} onEsc={onClose}>
+          <Box
+            fill="vertical"
+            overflow="auto"
+            width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+            pad="medium"
+            gap="medium"
+          >
+            <Header
+              direction="column"
+              align="start"
+              gap="xxsmall"
+              pad={{ horizontal: 'xxsmall' }}
+            >
+              <Heading level={4} size="small" margin="none">
+                Let us know how your expirence was!
+              </Heading>
+            </Header>
+            <Box
+              // Padding used to prevent focus from being cutoff
+              pad={{ horizontal: 'xxsmall' }}
+            >
+              <Form
+                method="post"
+                validate="submit"
+                kind="survey"
+                onSubmit={(value) => {
+                  console.log('submit', value);
+                  setIsSuccessful(true);
+                  closeFeedbackModal();
+                }}
+              >
+                <FormField
+                  htmlFor="thumbs-rating"
+                  name="rating"
+                  label="Was this content helpful?"
+                >
+                  <ThumbsRating
+                    id="thumbs-rating"
+                    name="rating"
+                    fillColor="pink"
+                    options={['1', '2']}
+                    color="pink"
+                  />
+                </FormField>
+                <FormField label="Comments" htmlFor="comments" name="comments">
+                  <TextArea id="comments" name="comments" />
+                </FormField>
+                {!isSuccessful ? (
+                  <Box
+                    margin={{ top: 'medium', bottom: 'small' }}
+                    direction="row"
+                    justify="end"
+                    gap="small"
+                  >
+                    <Button onClick={onClose} label="Cancel" />
+                    <Button label="Submit Feedback" primary type="submit" />
+                  </Box>
+                ) : (
+                  <Box margin={{ top: 'medium', bottom: 'small' }} align="end">
+                    <Text weight="bold" size="large">
+                      Thank you for your response!
+                    </Text>
+                  </Box>
+                )}
+              </Form>
+            </Box>
+          </Box>
+        </Layer>
+      )}
+    </>
+    // </Grommet>
+  );
+};
+
+UnSolicitedFeedback.storyName = 'UnSolicited feedback';
```

Comment:
I think we can exclude this story from having a chromatic snapshot

Reply:
done